components/Ui/UiLink.vue

```diff
@@ -1,12 +1,21 @@
 <template>
-  <UiBasicLink class="link" :class="`link_${kind}`" v-bind="$attrs">
+  <UiBasicLink
+    class="link"
+    :class="`link_${kind}`"
+    :segment="link?.segment"
+    :title="link?.title"
```

Comment:
The `UiBasicLink` component doesn't have a `title` property, so I think that we shouldn't pass this data.

```suggestion
```

Reply:
You are right! commit accepted!